packages/mars/src/options/chain/ethereum.ts

```diff
@@ -0,0 +1,60 @@
+import { ChainSet } from './ChainSet'
+
+export const networks: ChainSet = {
+  mainnet: {
+    chainId: 1,
+    chainName: 'Mainnet',
+    getPublicRPC: () => 'https://main-light.eth.linkpool.io/',
+    getInfuraRPC: (infuraApiKey) => `https://mainnet.infura.io/v3/${infuraApiKey}`,
+    getAlchemyRPC: (alchemyApiKey) => `https://eth-mainnet.alchemyapi.io/v2/${alchemyApiKey}`,
+    getEtherscanApi: () => 'https://api.etherscan.io/api',
+  },
+  ropsten: {
+    chainId: 3,
+    chainName: 'Ropsten',
+    getPublicRPC: () => 'https://ropsten.infura.io/v3/9aa3d95b3bc440fa88ea12eaa4456161',
+    getInfuraRPC: (infuraApiKey) => `https://ropsten.infura.io/v3/${infuraApiKey}`,
+    getAlchemyRPC: (alchemyApiKey) => `https://eth-ropsten.alchemyapi.io/v2/${alchemyApiKey}`,
+    getEtherscanApi: () => 'https://api-ropsten.etherscan.io/api',
+  },
+  rinkeby: {
+    chainId: 4,
+    chainName: 'Rinkeby',
+    getPublicRPC: () => 'https://rinkeby-light.eth.linkpool.io/',
+    getInfuraRPC: (infuraApiKey) => `https://rinkeby.infura.io/v3/${infuraApiKey}`,
+    getAlchemyRPC: (alchemyApiKey) => `https://eth-rinkeby.alchemyapi.io/v2/${alchemyApiKey}`,
+    getEtherscanApi: () => 'https://api-rinkeby.etherscan.io/api',
+  },
+  goerli: {
+    chainId: 5,
+    chainName: 'Goerli',
+    getPublicRPC: () => 'https://goerli-light.eth.linkpool.io/',
+    getInfuraRPC: (infuraApiKey) => `https://goerli.infura.io/v3/${infuraApiKey}`,
+    getAlchemyRPC: (alchemyApiKey) => `https://eth-goerli.alchemyapi.io/v2/${alchemyApiKey}`,
+    getEtherscanApi: () => 'https://api-goerli.etherscan.io/api',
+  },
+  kovan: {
+    chainId: 42,
+    chainName: 'Kovan',
+    getPublicRPC: () => 'https://kovan.infura.io/v3/9aa3d95b3bc440fa88ea12eaa4456161',
+    getInfuraRPC: (infuraApiKey) => `https://kovan.infura.io/v3/${infuraApiKey}`,
+    getAlchemyRPC: (alchemyApiKey) => `https://eth-kovan.alchemyapi.io/v2/${alchemyApiKey}`,
+    getEtherscanApi: () => 'https://api-kovan.etherscan.io/api',
+  },
+  arbitrum: {
+    chainId: 42161,
+    chainName: 'Arbitrum',
+    getPublicRPC: () => 'https://arb1.arbitrum.io/rpc',
+    getInfuraRPC: (infuraApiKey) => `https://arbitrum-mainnet.infura.io/v3/${infuraApiKey}`,
+    getAlchemyRPC: (alchemyApiKey) => `https://arb-mainnet.g.alchemy.com/v2/${alchemyApiKey}`,
+    getEtherscanApi: () => 'https://api.arbiscan.io/api',
+  },
+  'arbitrum-rinkeby': {
+    chainId: 421611,
+    chainName: 'Arbitrum Testnet',
+    getPublicRPC: () => 'https://rinkeby.arbitrum.io/rpc',
+    getInfuraRPC: (infuraApiKey) => `https://arbitrum-rinkeby.infura.io/v3/${infuraApiKey}`,
+    getAlchemyRPC: (alchemyApiKey) => `https://arb-rinkeby.g.alchemy.com/v2/${alchemyApiKey}`,
+    getEtherscanApi: () => 'https://api-testnet.arbiscan.io/api',
```

Comment:
I'd rename `getEtherscanApi` to sth like `getBlockExplorerApi`. Possibly even split to `getBlockExplorerApi` and `getContractVerifierApi`. Or just `getContractVerifierApi` and then pass the whole URL and not assume that each of these APIs works on exactly the same paths.

Reply:
It can be polished in a separate PR though.